orttraining/orttraining/python/training/ortmodule.py

```diff
@@ -287,7 +308,7 @@ def backward(ctx, *grad_outputs):
                     "The model's forward method has **kwargs parameter which is currently not supported.")
 
         self._onnx_inference = None
-        self._is_training = True
+        self.is_training = True
```

Comment:
self._is_training is not initialized at __init__?

Reply:
it is being initialized by the setter. It would defeat the purpose of setting it here since we want it to be only set in the setter method.

Reply:
Current implementatio sets it as True at https://github.com/microsoft/onnxruntime/blob/master/orttraining/orttraining/python/training/ortmodule.py#L290

A model is in training model by default. we dont have to call model.train() to put it training, right?

Reply:
I think this was set with the assumption that user model is in training mode. What we can do is, set the training mode to the training mode of the original user model. Or better still, not have a flag of our own. We just leverage the original_model training flag.
I can make these changes.